oauth2-useragent-core/src/main/java/com/microsoft/alm/oauth2/useragent/JavaFxProvider.java

```diff
@@ -14,7 +14,8 @@
     private static final List<String> REQUIREMENTS = Collections.unmodifiableList(Arrays.asList(
         "Oracle Java SE 7 update 6 or higher, Oracle Java SE 8, OR OpenJDK 8.",
         "JavaFX or OpenJFX runtime JAR.",
-        "A desktop environment."
+        "A desktop environment.",
+        "Oracle Java SE 7 is not supported on Mac OS X 10.11 and greater.  Please upgrade to Java 8."
     ));
     private final File[] potentialJavaFxJarLocations = new File[]{

```

Comment:
Child paths below look like absolute ones. Is that OK for all operating systems? The method description says: "... If the child pathname string is absolute then it is converted into a relative pathname in a system-dependent way. ..."